### DISCUSSION TITLE
Take a look on my Project . I'm new here with  less star

### DISCUSSION THREAD

**Discussion by mehulrajdev** [community/community#62934] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

https://github.com/mehulrajdev/Zayn-Malik

https://github.com/mehulrajdev

**Comment by yaelheerz** [comment]
cool!

**Comment by queenofcorgis** [comment]
Hi @mehulrajdev, we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by mehulrajdev** [reply, marked as answer]
ok
